### DISCUSSION TITLE
使用了 cz-git 后 是不是不需要再使用 commitlint/cli @commitlint/config-conventional"

### DISCUSSION THREAD

**Discussion by laterdayi** [Zhengqbbb/cz-git#112] [Q&A]
使用了 cz-git 后 是不是不需要再使用
"@commitlint/cli": "^17.4.3",
"@commitlint/config-conventional": "^17.4.4",

**Comment by Zhengqbbb** [comment]
他们可以作为本地 git hook 检测，但其实这个也可以写的简单脚本进行替换(e.g [verifyCommit](https://github.com/vuejs/core/blob/main/scripts/verifyCommit.js) )毕竟这两有点大，而且依赖下载老是报警告 
但是本地 git hook 本身就是不靠谱的，很容易屏蔽，真正严格的方式可以用 action 

https://packagephobia.com/result?p=@commitlint/cli,@commitlint/config-conventional
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/40693636/227076454-ac46c717-8611-4ddd-bebe-84f2670d4861.png">
